tests/integration_tests/performance/test_block_performance.py

```diff
@@ -0,0 +1,541 @@
+                IOPS_READ_STDDEV, IOPS_READ, st.function.Stddev),
+            st.consumer.StatisticDef(
+                BW_READ_MEAN, BW_READ, st.function.Avg,
+                criteria.EqualWith(baseline_bw_read["target"],
+                                   baseline_bw_read["target"] *
+                                   baseline_bw_read["delta_percentage"] /
+                                   100)),
+            st.consumer.StatisticDef(BW_READ_STDDEV, BW_READ,
+                                     st.function.Stddev),
+            st.consumer.StatisticDef(
+                SLAT_READ_MEAN, SLAT_READ, st.function.Avg,
+                criteria.EqualWith(baseline_slat_read["target"],
+                                   baseline_slat_read["target"] *
+                                   baseline_slat_read["delta_percentage"] /
+                                   100)),
+            st.consumer.StatisticDef(
+                SLAT_READ_STDDEV, SLAT_READ, st.function.Stddev),
+            st.consumer.StatisticDef(
+                CLAT_READ_MEAN, CLAT_READ, st.function.Avg,
+                criteria.EqualWith(baseline_clat_read["target"],
+                                   baseline_clat_read["target"] *
+                                   baseline_clat_read["delta_percentage"] /
+                                   100)),
+            st.consumer.StatisticDef(
+                CLAT_READ_STDDEV, CLAT_READ, st.function.Stddev),
+            st.consumer.StatisticDef(
+                LAT_READ_MEAN, LAT_READ, st.function.Avg,
+                criteria.EqualWith(
+                    baseline_lat_read["target"],
+                    baseline_lat_read["target"] *
+                    baseline_lat_read["delta_percentage"] / 100)),
+            st.consumer.StatisticDef(LAT_READ_STDDEV, LAT_READ,
+                                     st.function.Stddev)]
+    return [
+        st.consumer.StatisticDef(IOPS_READ_MEAN, IOPS_READ, st.function.Avg),
+        st.consumer.StatisticDef(IOPS_READ_STDDEV, IOPS_READ,
+                                 st.function.Stddev),
+        st.consumer.StatisticDef(BW_READ_MEAN, BW_READ, st.function.Avg),
+        st.consumer.StatisticDef(BW_READ_STDDEV, BW_READ, st.function.Stddev),
+        st.consumer.StatisticDef(SLAT_READ_MEAN, SLAT_READ, st.function.Avg),
+        st.consumer.StatisticDef(SLAT_READ_STDDEV, SLAT_READ,
+                                 st.function.Stddev),
+        st.consumer.StatisticDef(CLAT_READ_MEAN, CLAT_READ, st.function.Avg),
+        st.consumer.StatisticDef(CLAT_READ_STDDEV, CLAT_READ,
+                                 st.function.Stddev),
+        st.consumer.StatisticDef(LAT_READ_MEAN, LAT_READ, st.function.Avg),
+        st.consumer.StatisticDef(LAT_READ_STDDEV, LAT_READ,
+                                 st.function.Stddev)]
+
+
+def get_rw_stats(env_id, fio_id, cpu=None):
+    """Return read/write statistics."""
+    stats = get_read_stats(env_id, fio_id, cpu)
+    if cpu:
```

Comment:
Don't fully understand the differences between if cpu and if not cpu.  Can you please explain?